components/BackButton/BackButton.module.css

```
@@ -0,0 +1,25 @@
+  margin: 10px 5px;
+}
+
+.backButton:hover {
+  background-color: #3543dc;
+  color: white;
+}
+
+.triggerArea {
+  display: flex;
+
+  margin-top: 16px;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)